### ISSUE TITLE
🐛 [BUG] - Toast - 在有選擇公司的情況下彈出 END_OF_TRIAL 的吐司通知

### ISSUE THREAD

**Issue by TzuHanLiang** [CAFECA-IO/iSunFA#1850]
### Searched for similar issues

- [ ] YES, I have searched for similar issues.

### Time of occurrence

Today 3:30 PM

### Device

None

### OS

macOS

### Reproduction URL

https://www.isunfa.com/users/dashboard

### Reproduction steps

```markdown
不確定再現辦法
```

### Current Result

```markdown
會在有選擇公司的情況下彈出 END_OF_TRIAL 的吐司通知
```

### Expected Result

```markdown
會在有選擇公司的情況下不會彈出 END_OF_TRIAL 的吐司通知
```

**Comment by TzuHanLiang** [comment]
PR #1854 有解這個問題，需要知道再現辦法的同事 @RexBearIU ，重新測試

**Comment by RexBearIU** [comment]
2024/08/06 Looks good today.

We can close this bug.

**Comment by Luphia** [comment]
Fix with #1845
